### PR TITLE
[#3] Add Axon Server section

### LINKED ISSUE #3: Add Axon Server section
The main readme should contain an Axon Server section.

### PR DESCRIPTION
**Pull request by smcvb**
This pull request resolves #3 by adding a dedicated Axon Server section.
Furthermore, it reverts most outbound links to developer.axoniq.io/axon-server in favor of referring to the subsection.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Although the main repository, [Axon Framework](https://github.com/AxonFramework/
 * **[Axon Framework](#axon-framework)** 
 * **[Getting Started](#getting-started)** 
 * **[Receiving Help](#receiving-help)**
+* **[Axon Server](#axon-server)** - Purpose built Message router for commands, event, and queries, and an efficient Event Store
```
Comment (saratry) [inline profile/README.md:13]:
```suggestion
* **[Axon Server](#axon-server)** - Purpose-built Event Store and Messaging Platform for commands and queries
```

File: profile/README.md [hunk 2]
```diff
@@ -85,6 +86,28 @@ There are a couple of things to consider when you're traversing anything Axon:
...
+The message-driven nature of the framework becomes particularly useful when scaling the application or breaking it apart into distinct [microservice](https://developer.axoniq.io/microservices/overview).
+Furthermore, by Event Sourcing, you are future-proof to any shift and change of the models.
+
+Both benefits come with their own set of requirements.
+Firstly, we are inclined to use a distributed version of the `CommandBus`, `EventBus`, and `QueryBus` to break apart an application, considering their distinct routing requirements.
+Secondly, as Event Sourcing requires us to store events forever, the `EventStore` will grow indefinitely.
+
+[Axon Server](https://developer.axoniq.io/axon-server/overview) greatly simplifies these requirements by implementing these four infrastructure components.
+It is a dedicated message routing solution knowledgeable about the need to consistently route commands, stream events as fast as possible, and support the differing querying needs of each service.
+Next to message routing, Axon Server contains a storage engine designed for the specific purpose of storing events for event sourcing.
```
Comment (saratry) [inline profile/README.md:102]:
```suggestion
But most of all, Axon Server is an event store, a database specifically designed for storing events for event sourcing purpose.
```

File: profile/README.md [hunk 3]
```diff
@@ -194,9 +217,9 @@ The same logic applies to Sagas that are backed by streaming processors and stor
 
 Note that although you can use this extension to adjust MongoDB into an Event Store, we do not necessarily recommend this.
 During the lifecycle of this extension, we have noticed predicaments with how MongoDB behaves, causing the Event Store to act suboptimal from a performance perspective.
-We thus recommend either Axon Server (the highest level of performance for event storage) or an RDBMS of choice instead.
+We thus recommend either [Axon Server](#axon-server) (the highest level of performance for event storage) or an RDBMS of choice instead.
```
Comment (saratry) [inline profile/README.md:221]: I believe it is better to write event store in lowercase.

Comment (smcvb) [reply profile/README.md:221]: Makes sense, will do!!

File: profile/README.md (hunk 1 above)
Comment (smcvb) [reply profile/README.md]: Very good point. Replaced my text with yours.

**Review by CodeDrivenMitch — APPROVED** [review]

**Pull request merged**